pkgs/by-name/li/linien-client/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, fetchFromGitHub
+, linien-common
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "linien-client";
+  version = "0.8.0";
```

Comment:
They share source code and need to be updated at the same time, I think.
In such cases, you can inherit version and src from `linien-common`.

e.g. https://github.com/NixOS/nixpkgs/blob/e4ad989506ec7d71f7302cc3067abd82730a4beb/pkgs/development/python-modules/openllm/default.nix#L52